DP03_04b_Advanced_Phase_Curve_Modeling.ipynb

```diff
@@ -9,7 +9,7 @@
     "<br>\n",
```

Comment:
<p>Now that astropy.modeling is added does it need to be also explained in addition to sbpy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Nice catch! I forgot to remove it. I used that for my own testing, but ended up not using it. Will delete it.</p>